### ISSUE TITLE
[feature] Host an instance of standalone masto-fe and recommend it in clients list

### ISSUE THREAD

**Issue by tsmethurst** [superseriousbusiness/gotosocial#3619]
Iceshrimp has made available a standalone version of the Mastodon frontend client that works very nicely for GoToSocial instances, barring a couple 404s for API endpoints we haven't stubbed yet:

- https://iceshrimp.dev/iceshrimp/masto-fe-standalone <-- source
- https://masto-fe.iceshrimp.dev <-- running version

It might be nice if we host our own instance of the standalone frontend at something like masto-fe.gotosocial.org, and then add it to our "try these clients" list, as a way of easing the way for people who are used to Mastodon and want to try GtS.

**Comment by rkingett** [comment]
Does Glitch have something like this? Because the Mastodon front end has some embarrassing accessibility issues while logged in, such as the main navigation links, home, notifications, edit profile link, aren't labeled and appear as empty links and the search button isn't even labeled, and Glitch and even Hometown patched these up long ago!

**Comment by Fastidious** [comment]
It worth noting that that Masto-FE is not the front end on the current Mastodon version. It seems to be based on Mastodon version 3.5. People used to Mastodon interface might still find it awkward.

**Comment by tsmethurst** [comment]
@rkingett I think the standalone masto-fe is based on Glitch :) You can try the version hosted by iceshrimp now at https://masto-fe.iceshrimp.dev, logging in with either your Mastodon account or a GoToSocial account, and see how it is screenreader-wise. Would be good to have some feedback on that before we go recommending it :)

**Comment by rochacbruno** [comment]
@tsmethurst I tested it yesterday and I found it interesting that it has Local Only as an option but I tried and did not work.

https://masto-fe.iceshrimp.dev/@bruno/01JETZ3BK51GWJ61KY6ZAN5WG7 

Is that supported? as far as I understood the local-only marker for this client is adding the 👁️ emoji.

**Comment by tsmethurst** [comment]
Local only is supported in GtS but we use the api flag `local_only` rather than the emoji code. We'd probably have to lightly edit a fork of the client to get it to work 100% nicely with GtS :)

**Comment by fl4nn** [comment]
Another little something that masto-fe is missing is the ability to configure filters, likely because in Mastodon proper those are part of the Account settings.

I'm not saying the Gts fork would have to implement that but I though that should be kept in mind because I already had someone stumble on that and assume GtS itself doesn't have filters.

**Comment by tsmethurst** [comment]
Good point! Definitely needs a bit more fiddling before it's ready for the big time.

**Comment by tsmethurst** [comment]
This is now added :) And it's mentioned in the link to masto-fe that it's *experimental*. That's probably warning enough :) 

For folks wanting to contribute to the masto-fe standalone frontend, it's on codeberg, and pull requests are very welcome: https://codeberg.org/superseriousbusiness/masto-fe-standalone
